.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
   - stty cols 200
   ## need to fix above docker-engine install to not require sudo
   - sudo lc ci
+  - ./blackbox-test/build-examples.sh
```

Comment:
I should have noticed that. Thanks. I've changed it, and am re-running the build now.